cPP_DSC.adoc

```diff
@@ -3902,7 +3902,7 @@ This family defines requirements for individual Root of Trust services that the
 
 FPT_ROT_EXT.1 Root of Trust Services, requires the TSF to identify the specific Roots of Trust it provides.
 
-FPT_ROT_EXT.2 Root of Trust for Storage, requires the TSF to prevent unauthorized access to data associated with its Root of Trust for Storage.
+FPT_ROT_EXT.2 Root of Trust for Storage, requires the TSF to prevent unauthorized access to SDOs associated with its Root of Trust for Storage.
```

Comment:
Do we think the things that create the RoT are SDOs?

Reply:
That's what the SFR text currently says. This change is just the summary of the SFR in the extended component definitions.

Reply:
@woodbe this commit changes it from data to SDOs